### ISSUE TITLE
Feature request: Ability to return full documents in query instead of only document locations

### ISSUE THREAD

**Issue by DRK3** [decentralized-identity/confidential-storage#137]
Hi,

In Aries-Framework-Go we have an [EDV REST provider](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/storage/edv/restprovider.go) which is used alongside an "[EncryptedFormatter](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/storage/edv/encryptedformatter.go)" in order to allow us to translate between (k,v) pairs and encrypted documents in a way that allows it to conform to the storage interface there. 

Currently the spec only allows document locations to be returned and not the full documents themselves. This results in us having to make two REST calls per `Get` operation, which was so expensive that I actually [implemented an optional "extension" to our TrustBloc EDV implementation](https://github.com/trustbloc/edv/pull/162) in order to support returning full documents on a query. Right now I'm in the process of adding this optional capability to the Aries-Framework-Go EDV REST provider.

It would be very beneficial for our use case (and probably others) if the spec could support the ability to return full documents.

Thanks! Let me know if you have any questions.

**Comment by DRK3** [comment]
@OR13 I see that there's a new repo for the EDV spec at https://github.com/decentralized-identity/edv-spec. Can this issue be moved over?

**Comment by OR13** [comment]
yes, we will be reviewing all labeled issues and moving them on the next edv topic call for the wg

**Comment by DRK3** [comment]
@OR13 OK, cool! I guess pull requests will need to be manually re-created?

**Comment by OR13** [comment]
yes, sadly. however, if the PR is not reviewed on a call in short order, please ping @dmitrizagidulin @tplooker and advise them to follow the agreed process.
